package.json

```diff
@@ -24,7 +24,7 @@
     "intl": "1.2.5",
     "jail-monkey": "2.3.0",
     "jsc-android": "241213.1.0",
-    "mattermost-redux": "github:mattermost/mattermost-redux#e3d38bf92c5a8b5066c0a6252fb750ccd7a0bf98",
+    "mattermost-redux": "github:mattermost/mattermost-redux#818f7241e247f976abe768969d516e8dd5e325a5",
```

Comment:
this commit hash corresponds to the `release-5.18` branch and not master, master is `6533fd2acdd67af70049282bfd587be984f299c4`.

You need to have master -> master and release-1.26 -> release-5.18